server/datastore/mysql/migrations/tables/20211202092042_RemovePolicyHistory.go

```diff
@@ -0,0 +1,49 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211202092042, Down_20211202092042)
+}
+
+func Up_20211202092042(tx *sql.Tx) error {
+	_, err := tx.Exec("DROP VIEW policy_membership")
+	if err != nil {
+		return errors.Wrap(err, "drop table policy_membership_history")
+	}
+
+	_, err = tx.Exec("DROP TABLE policy_membership_history")
+	if err != nil {
+		return errors.Wrap(err, "drop table policy_membership_history")
+	}
+
+	policyMembershipTable := `
+		CREATE TABLE IF NOT EXISTS policy_membership (
+			policy_id INT UNSIGNED,
+			host_id int(10) UNSIGNED NOT NULL,
+			passes BOOL DEFAULT NULL,
+			created_at timestamp DEFAULT CURRENT_TIMESTAMP,
+			updated_at timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+			PRIMARY KEY (policy_id,host_id),
```

Comment:
So this is the same as policy_membership_history, but with this primary key, and then when we insert later on, the main change is the update on duplicate key.